### PR TITLE
Pattern Library: Convert to Bower Package - Take 2

### PR DESCRIPTION
**Pull request by talbs**
As an alternative to https://github.com/edx/ux-pattern-library/pull/141, this work converts the``_src/pattern-library`` into a bower package for other apps to use while maintaining the doc site and source code structure in this repo.

**A Few Notes**

* This does not move the PLDOC site to consume the pattern library source code as a Bower package
* This does not publicly publish the ``edx-pattern-library`` in the Bower registry. That will be done later (along with documentation).

- - -

#### Reviewers

* [x] **FED/Package Infrastructure** - @dsjen 
* [x] **README/Usage Documentation** - @marcotuts
* [x] **Open edX License and Contributing Standards** - @sarina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,111 +82,254 @@ brew install node
...
 
-**NOTE: some gulp tasks had to be separated to support both the pattern library 1)elements and 2)reference site UI separately.** Any task prefixed with `pldoc_*` refers to the reference site UI and not the elements.
+## Development - UX Pattern Library Bower Package
+
+If you add or edit any patterns in this repo (found in ``_src/pattern-library/``) or change any bower-centric settings (e.g. ``bower.json``), please do the following:
 
-### Publishing & Viewing Remotely
-The most up-to-date version of the library can be viewed on our public domain, **http://ux.edx.org**
+1. test that the change you've made will not negatively affect the bower package other apps and codebases are relying on
+2. talk to the UX Team about any bumps (based on [semantic versioning](http://semver.org/)) in the bower package's version number that are needed.
```
Comment (marcotuts) [inline README.md]: on this note - 1 nice to have might be an email address / other way to reach out in open use cases (or people could use the issues area?)

**Comment by marcotuts** [comment]
:+1: from my perspective.

**Comment by talbs** [comment]
@marcotuts, mind trying the following out as a last review?

1. Git clean your ux-pattern-library repo locally and check out this branch, run ``npm install``, ``bower install``, and ``gulp`` - the PL Doc Site should render locally as it always has
2. in a separate local folder outside of this work, create a test directory and run the following command to manually install a version of the PL bower package to it - ``bower install 'edx/ux-pattern-library#4f8798d10b027ffd342241213b21add57d2a2c29' --save``. You can then check out the directories/folders that the edX pattern library will install for you in a project.

**Comment by marcotuts** [comment]
@talbs - I was able to successfully install this bower package in a separate folder, which adds a bower_components folder and corresponding subfolders. 

Screenshot:
image

**Comment by talbs** [comment]
Thanks, @marcotuts!

**Pull request merged**